english/marshmallow.md

```diff
@@ -14,9 +14,7 @@ Grave danger.
 
 *(Is there another kind?)*
 
-The type of mortal danger that so often accompanies confectionary inspired interior design.
-Quickly surveying your immediate surrounds, you notice a garishly dressed, 
-vacant eyed young man lounging on a nearby sofa.
+The type of mortal danger that so often accompanies confectionary inspired interior design. Quickly surveying your immediate surrounds, you notice a garishly dressed, vacant eyed young man lounging on a nearby sofa.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.